mscore/scoreview.cpp

```diff
@@ -2802,6 +2802,10 @@ QSizeF ScoreView::fsize() const
       return QSizeF(s.width() * imatrix.m11(), s.height() * imatrix.m22());
       }
 
+constexpr qreal scrollStep   {   .8 };
+constexpr qreal thinPadding  { 10.0 };
+constexpr qreal thickPadding { 25.0 };
```

Comment:
Review needed for the two padding consts. I am not sure if the magic numbers are actually always padding in the code where they are used